surveys/2024-annual-survey/translations/de.md

```diff
@@ -712,8 +712,8 @@ Type: select one
 - Dänemark
 - Ecuador
 - Ein anderes Land
-- Elfenbeinküste
 - El Salvador
+- Elfenbeinküste
```

Comment:
Note that this does _not_ match the actual sort order shown on the survey (same for "Vereinigtes Königreich ..."). The previous version was actually matching the sort order of the survey.